astropy/logger.py

```diff
@@ -422,8 +422,6 @@ def log_to_list(self, filter_level=None, filter_origin=None):
 
         Parameters
         ----------
-        filename : str
```

Comment:
Good catch!

Reply:
Yeah, that's also one of the advantage of using `*args`, and `**kwargs` in docs, my tool will automatically fix typo and remove non existed documented parameter when it found some.